core/common/src/ECSqlReader.ts

```diff
@@ -342,6 +342,10 @@ export class ECSqlReader implements AsyncIterableIterator<QueryRowProxy> {
       return [];
     }
     const valueFormat = this._options.rowFormat === QueryRowFormat.UseJsPropertyNames ? DbValueFormat.JsNames : DbValueFormat.ECSqlNames;
+    if (this._options.rowFormat === QueryRowFormat.UseJsPropertyNames && typeof this._options.convertClassIdsToClassNames === "undefined") {
```

Comment:
can we please deprecate `QueryRowFormat.UseJsPropertyNames` ?

Reply:
fyi @ColinKerr 
deprecation doesnt have to be done in this specific pr, but this needs to be done asap